src/main/webapp/projectReview/ProjectReview.js

```diff
@@ -880,38 +916,38 @@ class ProjectReview extends Component {
           button({
             className: "btn buttonPrimary floatRight",
             onClick: this.submitEdit(),
-            disabled: this.compareObj("formData", "current") || this.compareObj("formData","editedForm"),
+            disabled: this.compareObj("formData", "current") || this.compareObj("formData", "editedForm"),
             isRendered: this.state.readOnly === false
           }, ["Submit Edits"]),
 
+          /*visible for Admin in readOnly mode and if the project is in "pending" status*/
+          button({
+            className: "btn buttonPrimary floatRight",
+            onClick: this.handleApproveInfoDialog,
+            disabled: this.state.disableApproveButton,
+            isRendered: this.isAdmin() && projectReviewApproved === 'false' && this.state.readOnly === true
```

Comment:
This is an example of why I prefer a typed boolean approach. This could (and probably should) be simplified to 
```suggestion
            isRendered: this.isAdmin() && !projectReviewApproved && this.state.readOnly
```